src/Coordination/RaftServerConfig.h

```diff
@@ -1,10 +1,13 @@
 #pragma once
+
+#include <vector>
```

Comment:
(minor, but the include order is usually:
- the own header file (in case it's a .cpp file)
- ClickHouse headers
- headers from third-party dependencies
- standard library headers like <vector> and <optional>

That ensures the least amount of transitive includes. So perhaps you like to move <vector> and <optional> back to their previous position.)

Reply:
Oh okay, I was moving them in the order of std includes, then our headers.